test/breedRoutes.test.js

```diff
@@ -13,14 +14,24 @@ const generateTestId = {
     
 const appTest = buildServer(generateTestId);
 
+
 //==================== user API tests ====================
 
+
 describe('GET /breeds', function () {
-    it('respond with json object containing a list of all breeds', function (done) {
+    it('respond with json object containing a list of all breeds - minimum 2 records to start', function (done) {
         request(appTest)
             .get('/breeds')
             .set('Accept', 'application/json')
             .expect('Content-Type', /json/)
+            .expect((res) => {
+                res.body.breeds[0].should.have.property('breedId', 1);
```

Comment:
```suggestion
                const firstBreed = res.body.breeds[0]
                firstBreed.should.have.property('breedId', 1);
```

Wherever possible it'd be good to use local variables when you're doing more than two things to access a value. This improves readability of the code which is really important when coming back to it later on.